templates/homepage/homepage_body.html

```diff
@@ -313,5 +313,5 @@ <h3 id="nosotrosh3" class="nosotrosh3">Visión:</h3>
 {% endblock %}
 
 {% block custom_js %}
-	<script async src="{% static 'homepage/js/homepage.js' %}"></script>
+	<script async src="{% static 'homepage/homepage.js' %}"></script>
```

Comment:
Just use `async` for js that do not depends on other js file, for example is this js file use jquery but as it's loading asyn i will load first than jQuery and not gonna find it and it's gonna break